test/JDBC/expected/BABEL-1000.out

```diff
@@ -123,15 +123,15 @@ SELECT * FROM babel_1000_test6(CAST('2030-05-06 13:59:29.123456 -8:00' AS dateti
 GO
 ~~START~~
 datetimeoffset#!#datetime2
-2030-05-06 13:59:29.120000 -08:00#!#1235-01-01 00:00:00.000000
+2030-05-06 13:59:29.1200000 -08:00#!#1235-01-01 00:00:00.0000000
```

Comment:
For all the tests where expected output is changing, it is a good practice to validate the results against SQL server. Framework provides easy way to run these tests against SQL server. This is specifically imp for such +-1 issues. Please ignore if already done.

Reply:
Yeah, I tested these changes against SQL Server already, and there is a small but intended difference between babelfish and SQL Server (last digit is always 0 for babelfish)

Reply:
I assume this difference is discussed, agreed upon and documented.

Reply:
Yes, this is the solution that was agreed upon with Bill and Rob.